media/css/base/page-not-found.scss

```diff
@@ -70,15 +70,17 @@ main {
     .logo-mozilla {
         display: inline-block;
         height: 50px;
-        transform: translateY(-6px);
+        width: 141px;
+        padding-left: 30px;
```

Comment:
This padding doesn't seem necessary and is causing the logo to misalign with the content.